### PR TITLE
[81X] Phase-1 tools: configuration updates

### PR DESCRIPTION
**Pull request by ghellwig**
Added some examples and updated slightly outdated configs.

### CODE REVIEW

**Comment by ghellwig** [comment]
please test

File: Alignment/APEEstimation/python/MisalignmentScenarios_cff.py [hunk 1]
```diff
@@ -1238,3 +1238,30 @@
         ),
     ),
 )
+
+MisalignmentScenario_PhaseI_PseudoAsymptotic = cms.PSet(
```
Comment (mmusich) [inline Alignment/APEEstimation/python/MisalignmentScenarios_cff.py]: @ghellwig please add a reference (twiki page) from where this numbers are taken from.

Comment (ghellwig) [inline Alignment/APEEstimation/python/MisalignmentScenarios_cff.py]: @mmusich do you mean to the documentation of this PR or to `MisalignmentScenarios_cff.py` itself?

File: Alignment/APEEstimation/test/Misalignments/createRandomlyMisalignedGeometry_cfg.py [hunk 2]
```diff
@@ -91,33 +111,36 @@
 scenariolabel = str(options.myScenario)
 if sigma > 0:
     scenariolabel = scenariolabel+str(sigma)
-outputfilename = "geometry_"+str(scenariolabel)+"_from"+process.GlobalTag.globaltag._value.replace('::All','')+".db"
+outputfilename = "geometry_"+str(scenariolabel)+"__from_"
+if options.inputDB is None:
+    outputfilename += process.GlobalTag.globaltag.value().replace("::All","")+".db"
```
Comment (mmusich) [inline Alignment/APEEstimation/test/Misalignments/createRandomlyMisalignedGeometry_cfg.py]: we don't support condDBv1 in 81x anymore, the `replace("::All","")` statement could be dropped

File: Alignment/APEEstimation/python/MisalignmentScenarios_cff.py (hunk 1 above)
Comment (mmusich) [inline Alignment/APEEstimation/python/MisalignmentScenarios_cff.py]: the documentation about the `MisalignmentScenario_PhaseI_PseudoAsymptotic`: I believe there is a twiki where the procedure how these numbers were derived is explained.

Comment (ghellwig) [inline Alignment/APEEstimation/python/MisalignmentScenarios_cff.py]: Yes, there ist a Twiki [1], but I wanted to know, if you would like to have this as a comment in the code fragment.

[1]
https://twiki.cern.ch/twiki/bin/view/CMS/TkAlPhaseIPreparations#Pseudo_asymptotic_scenario

Comment (mmusich) [inline Alignment/APEEstimation/python/MisalignmentScenarios_cff.py]: @ghellwig sorry, I realized I didn't answer your question: I mean add a reference in the code itself. 
Actually why is there a `Alignment/APEEstimation/python/MisalignmentScenarios_cff.py` file at all? 
Can we try to aggregate everything here: https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/Alignment/TrackerAlignment/python/Scenarios_cff.py where previous scenarios were documented?

Comment (ghellwig) [inline Alignment/APEEstimation/python/MisalignmentScenarios_cff.py]: Sure, can be done!
I think also the configuration file should be moved away from APEEstimation to TrackerAlignment.
Will do that to clean things up a bit.

**Comment by ghellwig** [comment]
please test

**Comment by mmusich** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**